src/middleware.ts

```diff
@@ -68,6 +68,12 @@ import { serverTimingHeaders } from "./lib/middleware/serverTimingHeaders"
 import { splitTestMiddleware } from "./desktop/components/split_test/splitTestMiddleware"
 import { IGNORED_ERRORS } from "./lib/analytics/sentryFilters"
 import { sharifyToCookie } from "./lib/middleware/sharifyToCookie"
+import { featureFlagMiddleware } from "lib/middleware/featureFlagMiddleware"
```

Comment:
since our feature flags is basically a lib, and we have `lib/featureFlags`, why not co-locate everything there so that related files aren't spread throughout the codebase?